pkg/socketcan/fileconn.go

```diff
@@ -30,8 +30,7 @@ type fileConn struct {
 	ra net.Addr
 }
 
-// ensure net.Conn interface
-var _ net.Conn = &fileConn{}
+var _ = net.Conn(&fileConn{})
```

Comment:
The previous form is generally seen as a more idiomatic type of assertion in Go. Any specific reason for the change?

Reply:
Interesting! I've been seeing this form more than the other form when reading various code, but I'll change back

Reply:
The previous form is asserting type, the latter is type converting it. It seems type assertion was the intended idea here but I won't claim to understand the technical differences between them under the hood